Gems/AWSClientAuth/gem.json

```diff
@@ -3,7 +3,8 @@
     "display_name": "AWS Client Authorization",
     "license": "Apache-2.0 Or MIT",
     "license_url": "https://github.com/o3de/o3de/blob/development/LICENSE.txt",
-    "origin": "Amazon Web Services, Inc.",
+    "origin": "Open 3D Engine - o3de.org",
```

Comment:
@lmbr-pip should the origin for the AWS gems be O3DE or AWS?

Reply:
Amazon owns the 3rdParty AWS native SDK but I don't think they own this engine gem which uses it. If Amazon owns it I dont think it would be distributed with the linux foundation open source project with copyrights that say:
 * Copyright (c) Contributors to the Open 3D Engine Project.
 * For complete copyright and license terms please see the LICENSE at the root of this distribution.
 * SPDX-License-Identifier: Apache-2.0 OR MIT

Thats not Amazon... so I think Amazon own the gem. I have asked Amazon legal for clarification, but I'm pretty sure it isn't.

Reply:
Reverted